README.md

```diff
@@ -92,9 +87,9 @@ The deep learning framework of the source code. For now, we support `pytorch` an
 Can be `null` if the implementation is not framework specific.\
 `language` and `framework` define which model runner can use this model for inference. 
 
-- `source`
+- `[source]`
```

Comment:
here, it's only optional depending on `weights_format`. should we indicate this differently? `[...]*` *depends on `other_field` ?

Reply:
Yeah, I think `[...]*` is good. (And in that case we should always explain it more in the description like here.)